frontend/src/components/auth/login/login.css

```diff
@@ -77,10 +84,14 @@
 }
 
 .login-input input:focus {
-  border-color: #1863ff !important ;
+  /* border-color: #1863ff !important ; */
```

Comment:
remove this commented line

Reply:
@Kajol-Kumari I did all the changes suggested by you. Also, before commenting all that code, I also did the same thing which you suggested but it was not working. Also, After doing changes you said, now also its not working from my end. Please check.

Thank you